### ISSUE TITLE
Just use newlines-between and ignore the import order

### ISSUE THREAD

**Issue by reinos** [import-js/eslint-plugin-import#2839]
Hi, 

I just want check if my import statement have no empty line in between. I do not care about the order of the import statements.

I was trying to do something like: 

```
rules:
  import/order:
    - "error"
    - "newlines-between": "never"
      "groups":
```

to enforce no empty lines in the import block but do not trigger the order sorting. However, the following error is then triggered. `Incorrect configuration of the rule: Unknown type "" eslint[import/order](https://github.com/import-js/eslint-plugin-import/blob/v2.27.5/docs/rules/order.md)`

Is there a way to have just an eslint rule that will check if there is no empty line in the import block?

**Comment by ljharb** [comment]
I don't use yaml config ever, so i don't know how to do it, but if it's an empty array instead of an empty string, it might work?

**Comment by reinos** [comment]
Empty array is working as expected :-)

```
rules:
  import/order:
    - "error"
    - "newlines-between": "never"
      "groups":
          -  []
```
